jans-lock/cedarling/tests/fetch-schema.rs

```diff
@@ -1,8 +1,14 @@
-use cedarling::utils;
+use cedar_policy::*;
 use wasm_bindgen_test::*;
 
 #[wasm_bindgen_test]
 async fn test() {
-	let schema = utils::fetch_schema().await;
+	let schema = include_str!("../policy-store/schema.txt");
+	let (schema, warnings) = Schema::from_str_natural(schema).unwrap();
+
+	for warning in warnings {
+		console_log!("Schema Parser generated Warning: {}", warning);
+	}
+
 	console_log!("Fetched Schema: {:?}", &schema)
```

Comment:
Needs to test via assertions

Reply:
Schema is parsed from the PolicyStore JSON, and that process doesn't emit any warnings sadly